modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/util/PowwowUtil.java

```diff
@@ -102,6 +103,13 @@
 		return calendarBookingString.getBytes();
 	}
 
+	public static long getDurationInMinutes(Calendar startTime, Calendar endTime) {
```

Comment:
we may not need to use this method. as in Powwow portlet, mostly we will display start and end time. Only need duration to push to Zoom API.